calcom/cal.com
# fix: only one webhook per event type

## What does this PR do?
Fixes #14919 

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
1. Create an event.
2. Open the event settings and go to the Webhooks tab and create a webhook.
3. Now you'll see a 'New Webhook' button to create a new webhook.

## Mandatory Tasks

- [*] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
